tests/unittests/Foundation/NSURLCacheTests.mm

```diff
@@ -63,7 +66,7 @@ static void _addFakeCacheResponse(NSURLCache* cache, NSCachedURLResponse* respon
     EXPECT_EQ(0, [cache currentMemoryUsage]);
 }
 
-TEST(NSURLCache, Promotion) {
+OSX_DISABLED_TEST(NSURLCache, Promotion) {
```

Comment:
>OSX_DISABLED_TEST [](start = 0, length = 17)

nit: provide similar comment as above or just mention that this test behaves similarly.